src/main/java/de/tum/in/www1/artemis/service/connectors/localci/buildagent/LocalCIDockerService.java

```diff
@@ -60,6 +62,9 @@ public class LocalCIDockerService {
     @Value("${artemis.continuous-integration.image-cleanup.expiry-days:2}")
     private int imageExpiryDays;
 
+    @Value("${artemis.continuous-integration.image-cleanup.disk-space-threshold-mb:1300}")
```

Comment:
I believe we need more, at least 2 GB. Some docker images are even larger..

Reply:
updated it to 2GB